Curtains/Curtains.Domain/Models/SetModel.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Curtains.Domain.Models
+{
+    public class SetModel : BaseEntity
```

Comment:
what is the set model?
i can't understand what this model about

Reply:
Set of products like curtains and pillows and etc. in one order

Reply:
Imagine that your are a new dev in this project and you see this model name. Would you correctly understand what this model describes?

Reply:
Understood